### PR TITLE
fix #41309 liveam.tv

### LINKED ISSUE #41309: liveam.tv
### Issue URL (Incorrect Blocking)
[https://liveam.tv/adskaya-kuhnya-25-09-2019.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fliveam.tv%2Fadskaya-kuhnya-25-09-2019.html)
### Comment
Username: @Riko-Kafto
> При включенном Adguard не работает плеер

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.2.1
Browser: | Yandex Browser
Filters: | AdGuard Russian,<br>Filter unblocking search ads and self-promotion,<br>EasyList Italy,<br>AdGuard Turkish,<br>AdGuard French,<br>AdGuard Base,<br>Official Polish filters for AdBlock, uBlock Origin & AdGuard,<br>AdGuard Tracking Protection,<br>AdGuard German,<br>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/41309

### CODE REVIEW

**Review by BlazDT — APPROVED** [review]

File: FrenchFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -8,6 +8,8 @@ comandotorrents.com#@#a[href^="http://www.masterads.info/"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/39013
 @@||dl.opensubtitles.org/*/download^
 @@||api.opensubtitles.org^$third-party
+! https://github.com/AdguardTeam/AdguardFilters/issues/41309
+@@||out.pladform.ru/player$domain=liveam.tv
```
Comment (Alex-302) [inline FrenchFilter/sections/whitelist.txt]: which filter blocks it?

Comment (zloyden) [reply FrenchFilter/sections/whitelist.txt]: `AdGuard French`

Comment (BlazDT) [reply FrenchFilter/sections/whitelist.txt]: This rule here: ||out.pladform.ru^.
Rule comes from Liste FR.

Comment (Alex-302) [reply FrenchFilter/sections/whitelist.txt]: this is a bad rule. out.pladform.ru should not be blocked completely by general rule. Better exclude it.

**Pull request closed without merging**